frontend/src/metabase/auth/selectors.ts

```diff
@@ -1,12 +1,9 @@
 import { createSelector } from "reselect";
+
+import { getSettings } from "metabase/selectors/settings";
 import { PLUGIN_AUTH_PROVIDERS } from "metabase/plugins";
-import { State } from "metabase-types/store";
-import { AuthProvider } from "./types";
 
-export const getSettings = createSelector(
-  (state: State) => state.settings,
-  settings => settings.values,
-);
+import type { AuthProvider } from "./types";
```

Comment:
Just removing the custom `getSettings` here and replacing it with an identical selector from `metabase/selectors/settings`, so it can be used by `getAuthProviders` down below. `getSettings` here is exported